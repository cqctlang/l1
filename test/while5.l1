i = 10;
while(--i > 0)
	if(i == 5)
		break;
print(i);
