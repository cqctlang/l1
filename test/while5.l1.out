entry: 0 params, 0 locals, 0 temps
<cval 5>
halted (ac = <nil>)
